src/components/textarea/_macro.njk

```diff
@@ -68,7 +68,6 @@
             "id": params.fieldId,
             "classes": params.fieldClasses,
             "dontWrap": params.dontWrap,
-            "legendIsQuestionTitle": params.legendIsQuestionTitle,
```

Comment:
I think this is still needed for mutually exclusive when not used in a question macro

Reply:
I think thats for onsFieldset not onsField. Ive left it in for onsFieldset